scripts/Database-Update.ps1

```diff
@@ -1,6 +1,6 @@
 Push-Location -Path $PSScriptRoot\..\backend\YouTubeV2.Application
 
 dotnet ef database update --startup-project ..\YouTubeV2
-dotnet ef database update --connection "Server=localhost;Integrated Security = true;Database=YouTubeV2Test;TrustServerCertificate=True" --startup-project ..\YouTubeV2
```

Comment:
I think you might have leaked your password. Don't change this btw if it doesnt work change your mssql settings